src/arena.py

```diff
@@ -1 +1,108 @@
-WORKS = True
+import json
+import re
+import random
+from typing import List
+
+
+class Damage:
+    __x: int
+    __y: int
+    __z: int
+
+    def __init__(self, damage: str):
+        # XdY[ + Z]
+        m = re.search(r"^(\d+)d(\d+)( \+ (\d+))?$", damage)
+        if m:
+            self.__x = int(m.group(1))
+            self.__y = int(m.group(2))
+            z = m.group(4)
+            if z:
+                self.__z = int(z)
+            else:
+                self.__z = 0
+
+    def draw(self):
+        result = 0
+        for i in range(self.__x):
+            result += random.randint(1, self.__y)
+        result += self.__z
+        return result
+
+
+class Combatant:
+
+    name: str
```

Comment:
You don't need to put here fields you set in `__init__`

Reply:
ok